src/lib/withFallbackImage.js

```diff
@@ -1,47 +1,64 @@
 import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import { defaultImage } from '../constants/collectives';
 import { getDomain, imagePreview } from './utils';
 
-export default ChildComponent => ({ type = 'USER', radius, height, website, name, ...props }) => {
-  const [src, setSrc] = useState(props.src);
-  if (name === 'anonymous') {
-    type = name.toUpperCase();
-  }
+const withFallbackImage = ChildComponent => {
+  const wrapped = ({ type = 'USER', radius, height, website, name, ...props }) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Why is this one disabled? Is there no way to make it work?

Reply:
I disabled the hooks rules warning, but it's working fine.